src/typescript/06-types/types.md

```diff
@@ -433,202 +596,52 @@ function getTotalDistanceAbleToTravel(dinos: Dinosaur[]): number {
 
 > **Note:** notice that we never check for `nextDino.type` to be `land`. We don’t have to! TypeScript is smart enough to figure out that there are only three types land, water, and air and since we have checked for the first two (air and water) it knows if it gets through those conditionals the type is land.
 
-### Enum
+### Setup 3
 
-Enums allow the aliasing of names to a list of numeric values. Like most indexing, enums start their first member at 0.
+✏️ Create the following files:
```

Comment:
We want to follow the same format as the others: https://www.bitovi.com/academy/learn-typescript/exporting-and-importing.html#setup

[The same line can be repeated over and over with just the filename different above each block.]